src/tests/__tests__/[...params].test.js

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import HomePage, { getServerSideProps } from '../../pages/company/[...params]';
```

Comment:
simply add in tests folder remove__tests__

Reply:
Done